homework/02LessonTwo.js

```diff
@@ -126,7 +159,7 @@ function whichSpecies(character) {
 //
 // In your own words, explain what conditionals do.
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: Conditionals evaluate to true or false of a block of code though of series of conditionals (e.g. if, else).
 //
```

Comment:
Yep! I accept this answer

![finnn](https://media.giphy.com/media/dZTOPFFtsISM8/giphy.gif)